script/thrift_gen.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
We should also add an entry for this in the [Makefile](https://github.com/open-telemetry/opentelemetry-php/blob/main/Makefile#L27)

Reply:
This file also needs executable permissions added to it so we can generate thrift output.  `chmod +x script/thrift_gen.sh` should handle this for you.

Reply:
- executable permission given to script/thrift_gen.sh
- Makefile execution failing due to [Dockerfile](https://github.com/open-telemetry/opentelemetry-php/blob/127166bc29f9fb7bf96ab5250a6118b7585d5e2f/docker/Dockerfile#L1) PHP:8 version as [apache/thrift](https://packagist.org/packages/apache/thrift) latest verion is compatible with php: ^5.5 || ^7.0 
- I modified  Dockerfile locally to use php:7.3-buster and it worked fine.

Let me know your thought on this, as  apache/thrift is not supporting PHP:8

Reply:
We talked about this in the SIG meeting today.  We are going to remove the hard dependency on PHP8 for now.  There is an apache thrift issue open about this:

https://github.com/apache/thrift/pull/2288

Once this issue has been closed, we can re-enable PHP8 as a dependency.

You can feel free to change this dependency for the time being:

* Change the dockerfile to use php:7.4-buster: https://github.com/open-telemetry/opentelemetry-php/blob/main/docker/Dockerfile#L1
* remove php 8.0 from the github action workflow : https://github.com/open-telemetry/opentelemetry-php/blob/main/.github/workflows/php.yml#L17

Reply:
@saktib - once you complete this, I think we'll be able to merge this?

Reply:
Yes, absolutely. I'll commit the above fixes soon.